src/main/java/com/microsoft/sqlserver/jdbc/SQLServerStatement.java

```diff
@@ -1368,7 +1368,7 @@ final void clearLastResult() {
      *
```

Comment:
Modify javadoc now that clearFlag is a parameter

Reply:
Done